build-rpi-raspbian/openhabian.raspbian.conf

```diff
@@ -17,4 +17,13 @@ system_default_locale="en_US.UTF-8"
 wifi_ssid=""
 wifi_psk=""
 
+# Governments assert the right to regulate usage of radio spectrum within
+# their respective territories so make sure you select a ISO/IEC 3166-1 alpha2
+# country code suitable for your location or you may infringe on local
+# legislature. See `/usr/share/zoneinfo/zone.tab' 
+# For a table of timezone descriptions containing ISO/IEC 3166-1 alpha2 country codes.
+# See https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2 
+# eg: US DE AU NZ...
+wifi_country=""
```

Comment:
How about we reduce this to a simple one/two-liner? Most details are irrelevant for the end user. We just need the "local legislature", wikipedia article and examples. Also mention US default

Reply:
I've replaced the empty string with US, which will effectively make it default